.github/workflows/run-dev.yml

```
@@ -27,6 +27,7 @@ jobs:
         run: python -m permanent_upload dev files/
   notify:
     runs-on: ubuntu-latest
+    if: always()
```

Do these need to be 
```suggestion
    if: ${{ always() }}
```

like in the docs?